packages/rps/src/core/channel-state.ts

```diff
@@ -10,6 +10,8 @@ export interface ChannelState<T = AppData> {
   bUserId: string;
   aAddress: string;
   bAddress: string;
+  aOutcomeAddress: string;
+  bOutcomeAddress: string;
```

Comment:
Could  we do better than `address`, `outcomeAddress` -- the former is ambiguous if you don't see it next to the latter.

Here we have https://app-wallet-interface.netlify.com/#introduction `signingAddress` and `destination`, and we should probably enforce consistency with a change here or there or both. The same consistency comment applies to the `nitro-protocol` package and docs website. 

Other possibilities: 
-  `channelAccount` and `chainAccount` 
-  `channelAddress` and `chainAddress`
-  `channelAddress` and `EOA` (externally owned account)
- `ephemeralAddress` and `outcomeAddress`

Reply:
I had initially used `participantAddress` and `outcomeAddress` but there was a difficult to find bug from changing `address` to `participantAddress` somewhere in the code. To save time on the weekend, I kept the former as `address`.

Reply:
Cool -- let's just punt this into an issue then, as this PR needn't be held up for this.

Reply:
https://github.com/statechannels/monorepo/issues/828